### ISSUE TITLE
auto-crossover detection not working in my case where I've renamed my CrossOver app and it's in a sub-folder

### ISSUE THREAD

**Issue by vesper8** [phoenixlauncher/phoenix#92]
I have multiple versions of CrossOver.. as this is the way that many macgaming influencers suggest to do so that you can have one that's been patched with CXPatcher and one that hasn't.. and so you can also use older versions in case of regressions.

```
/Applications/_GAMING_WINDOWS
/Applications/_GAMING_WINDOWS/CrossOver 23.6 CXPatcher.app
/Applications/_GAMING_WINDOWS/CrossOver 23.6.app
/Applications/_GAMING_WINDOWS/CrossOver 23.7.app
```

As it is.. I checked the option to "auto detect crossover games" and browsed to /Applications/_GAMING_WINDOWS, from here I couldn't select any of my 3 custom-named CrossOver apps. Just selecting the folder seemed to have no effect and no games were detected after I closed and re-opened the launcher.

I also tried renaming one back to the standard "CrossOver.app" but I also couldn't select it.

There should probably be a "Auto-detect Now" kind of functionality so you don't have to restart the launcher..

It would be nice if you could just select multiple locations where to auto-detect games too. Maybe instead of picking the CrossOver.app you should allow us to provide the full path to the bottles themselves. CXPatcher has a toggle for using a different paths for its bottles so that they don't conflict with the unpatched version.

Also I noticed that when there are no games at all and there's only the placeholder game, trying to delete the placeholder game just crashes the app.

Anyway it's great to see you're actively working on this. Looking forward to using this in the future as a replacement for CrossOver's awful UI.

**Comment by jxhug** [comment]
Sorry if it's unclear, this is for selecting the folder containing apps/bottles, not CrossOver installations.

**Comment by vesper8** [comment]
@jxhug Hrm yes.. quite unclear.. specially since I could have sworn that the placeholder included CrossOver.app

The path to the bottles doesn't. In my case it's

`/Users/redacted/Library/Application\ Support/CrossOver`

This is the folder that contains the bottles.

I tried to give it that folder, saved, closed the app, re-opened.. nothing

Then I tried to give it `/Users/redacted/Library/Application\ Support/CrossOver/Bottles`

Saved, re-opened.. still nothing

What am I doing wrong?

Also.. I noticed that if I try to Browse to this folder, the one inside Library/Application Support.. the browse window does not show me the Library folder at all.. this despite me having all hidden/system folders not be hidden because I'm a developer and I don't like hidden files and folders.. and usually it is not hidden in "browse" windows.. but in yours.. it is.. so I'm thinking.. if you expect users to manually provide you with this path.. it's going to be real hard. I had to open that folder in a separate Finder window and drag it into the browse window in order to be able to set it.

**Comment by vesper8** [comment]
Thanks. That worked, it detected a bunch of games.

I also tried to have it detect my steamapps folder and I kept getting an error saying that the path wasn't valid and must end with steamapps.. but i tried giving it both the steamapps folder as well as the parent containing folder where the steamapps folder is.. and it wouldn't accept either.

My steamapps folder is `/Volumes/Games/SteamLibrary/steamapps`

**Comment by vesper8** [comment]
Btw I just tried launching a game which isn't on my Steam, just an exe I set up with a command launcher in CrossOver.. and upon trying to launch the game, instead of launching the exe it launched Steam and gave an error. I can launch the game fine in CrossOver.

Phoenix should not make assumptions.. if the CrossOver launcher points to an exe directly it should use that, not assume the game is on Steam.

**Comment by jxhug** [comment]
Yeah sorry, that's currently being looked at in #91

**Comment by jxhug** [comment]
> Thanks. That worked, it detected a bunch of games.
> 
> I also tried to have it detect my steamapps folder and I kept getting an error saying that the path wasn't valid and must end with steamapps.. but i tried giving it both the steamapps folder as well as the parent containing folder where the steamapps folder is.. and it wouldn't accept either.
> 
> My steamapps folder is `/Volumes/Games/SteamLibrary/steamapps`

<img width="340" alt="image" src="https://github.com/phoenixlauncher/phoenix/assets/72482214/2ce3f8e7-6374-4a05-863d-22d9f785cc67">

that's odd. i have a very similar structure to you and mine works. will look into this

**Comment by jxhug** [comment]
i've found the bug. very simple, will be fixed in the next pr when I fix #91
